pkg/handlers/generic/lifecycle/ccm/nutanix/handler_test.go

```diff
@@ -209,5 +209,7 @@ func moduleRootDir() string {
 				out),
 		)
 	}
-	return strings.TrimSpace(string(out))
+	modules := strings.Split(string(out), "\n")
+	rootModule := strings.TrimSpace(modules[0])
+	return rootModule
```

Comment:
Alternatively use `strings.Cut` (example at https://play.golang.com/p/i5_LmeCTlA8).

```suggestion
	rootModule, _, _ := strings.Cut(string(out), "\n")
	return rootModule
```

Reply:
Looks like `Cut` was added in go 1.18, but I missed it. Thanks!

Reply:
Fixed in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/1125/commits/74076ab0fd88a746d2bb2564ad054402d94eada9